src/elements/dialogs/session-details.html

```diff
@@ -210,6 +214,19 @@ <h3>{$ sessionDetails.speakers $}</h3>
       _getFeaturedSessionIcon(featuredSessions, sessionId) {
         return featuredSessions && featuredSessions[sessionId] ? 'bookmark-check' : 'bookmark-plus';
       }
+
+      _findSpeakers(speakers) {
+        if (speakers == null) {
+          return [];
+        }
```

Comment:
if (!speakers) return [];

Reply:
Done. 

But, to let you know, this kind of syntax leads to 6 times more instructions generated by V8. This also could lead to bugs if, for some reason (we are in JS), the value became the `number` 0.